src/JiraStatusNotifier/Domain/JiraConnector.php

```diff
@@ -14,6 +14,7 @@
 use Chemaclass\JiraStatusNotifier\Domain\Jira\ReadModel\Company;
 use Chemaclass\JiraStatusNotifier\Domain\Jira\TicketsByAssignee\StrategyFilter\TicketFilter;
 use Chemaclass\JiraStatusNotifier\Domain\Jira\TicketsByAssignee\TicketsByAssigneeClient;
+use Chemaclass\JiraStatusNotifier\Domain\Jira\TicketsByAssignee\TicketsByAssignee;
```

Comment:
You got this right, but there is one tiny detail here. The coding guidelines are failing because the uses are not in order. See: https://github.com/Chemaclass/JiraStatusNotifier/runs/4993238464?check_suite_focus=true
```
Loaded config default from "/home/runner/work/JiraStatusNotifier/JiraStatusNotifier/.php-cs-fixer.dist.php".
.......................F...................................       59 / 59 (100%)
   1) JiraStatusNotifier/src/JiraStatusNotifier/Domain/JiraConnector.php (ordered_imports)
```
You can identify these errors by running `composer test-all`, for example, and it will check everything (see composer.json line 62). To fix all style errors run `composer csfix`.

Reply:
I see you fixed it before I even finished my comment. Nice! 👌

Reply:
Thanks for pointing that. I'll fix